script/hassfest/services.py

```diff
@@ -262,6 +262,23 @@ def validate_services(config: Config, integration: Integration) -> None:  # noqa
                     f"Service {service_name} has no description {error_msg_suffix}",
                 )
 
+        if integration.core and "services" in strings:
+            section_fields = set()
+            for content in service_schema.get("fields", {}).values():
+                if "fields" in content:
+                    # This is a section
+                    section_fields.update(content["fields"].keys())
+            fields = {
```

Comment:
```suggestion
            translation_fields = {
```
just to make it easier to understand for the var names